libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/theme/components/Checkbox.kt

```diff
@@ -46,6 +49,16 @@ fun Checkbox(
     )
 }
 
+@Composable
+internal fun compoundCheckBoxColors(): CheckboxColors {
+    return CheckboxDefaults.colors(
+        uncheckedColor = ElementTheme.colors.borderInteractivePrimary,
+        disabledUncheckedColor = ElementTheme.colors.borderDisabled,
+        disabledCheckedColor = ElementTheme.colors.iconDisabled,
+        disabledIndeterminateColor = ElementTheme.colors.iconDisabled,
+    )
```

Comment:
Shouldn't we use all the possible parameters here? `checkedColor` and `checkmarkColor` are missing.

Reply:
Those will use the default tokens. We can be explicit and set those too though.

Reply:
OK, thanks. Maybe keep it like this, that's fine.